test/GeneratedInjectorDelegatorTest.php

```diff
@@ -27,7 +28,7 @@ public function testProvidedNamespaceIsNotAString()
 
         $delegator = new GeneratedInjectorDelegator();
 
-        $this->expectException(ContainerExceptionInterface::class);
+        $this->expectException(InvalidServiceConfigException::class);
```

Comment:
it was quite important check, as container (per specification) can throw only `ContainerExceptionInterface` instances.

would be nice to have it somewhere

Reply:
Is a unit test really the correct scope? We could assert that `InvalidServiceConfigException` implements `ContainerExceptionInterface`. So that whenever this changes this test fails:

```suggestion
        self::assertInstanceOf(ContainerInterface::class, new InvalidServiceConfigException());
        $this->expectException(InvalidServiceConfigException::class);
```

Reply:
Yeah, probably it should be separate test case, but I am happy to have it here for time being.

Reply:
I guess a separate test would be out of scope for now. I'm not entirely sure if this is the correct place or way to cover this. Maybe we find something more appropriate in the future. :wink: 

Until then I added the assert as suggested.